packages/account-v2/src/containers/POAForm/AddressDetailsForm.tsx

```diff
@@ -0,0 +1,127 @@
+        updateSettings({
+            address_city: values.addressCity,
+            address_line_1: values.addressLine1,
+            address_line_2: values.addressLine2,
+            address_postcode: values.addressPostcode,
+            address_state: values.addressState,
+        });
+
+        // upload file
+        return upload({
+            document_issuing_country: settings?.country_code ?? undefined,
+            document_type: 'proofaddress',
+            file: values.document,
+        });
+    };
+
+    const { address_city, address_line_1, address_line_2, address_postcode, address_state } = settings;
+
+    const initialValues: TAddressDetails = {
+        addressCity: address_city ?? '',
+        addressLine1: address_line_1 ?? '',
+        addressLine2: address_line_2 ?? '',
+        addressPostcode: address_postcode ?? '',
+        addressState: address_state ?? '',
+    };
+
+    if (fetchError) {
+        return <POAError error_message={fetchError.error.message} />;
+    }
+
+    const formSubmitError = settingsUpdateError?.error.message || documentUploadError?.error.message;
+
+    return (
+        <Fragment>
+            {isMobile && (
+                <div className='grid grid-cols-[auto_25px] items-center pb-300 mb-500 border-solid border-b-75 border-solid-grey-2'>
+                    <Text align='center' size='lg' weight='bold'>
+                        Proof of address
+                    </Text>
+                    <StandaloneXmarkBoldIcon
+                        iconSize='md'
+                        onClick={() => {
+                            history.push('/account-v2/');
+                        }}
+                    />
+                </div>
+            )}
+            <Formik enableReinitialize initialValues={initialValues} onSubmit={handleFormSubmit} validateOnMount>
+                {({ isSubmitting, isValid }) => (
+                    <Form>
+                        <div className='flex flex-col w-full min-h-screen sm:w-auto space-y-800'>
+                            {(formSubmitError || resubmitting) && (
+                                <InlineMessage size='md' type='error' variant='contained'>
+                                    {formSubmitError ||
+                                        `We were unable to verify your address with the details you provided. Please check
+                                    and resubmit or choose a different document type.`}
+                                </InlineMessage>
+                            )}
+                            <div className='overflow-y-auto m-50 space-y-600'>
+                                <div className='flex h-1200 gap-400 self-stretch sm:self-auto justify-center items-center sm:gap-[11px]'>
+                                    <Text weight='bold'>Address</Text>
+                                    <div className='w-full h-75 flex-[1_1_0] bg-solid-grey-2 sm:flex-shrink-0' />
```

Comment:
We need not give `sm`, as Tailwind is mobile first.

Reply:
this sm breakpoint refers to the screens above mobile. 
https://tailwindcss.com/docs/responsive-design#targeting-mobile-screens